### PR TITLE
Update README.md with some description

### PR DESCRIPTION
**Pull request by vejeta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
-Testing different methods to read from SFTP from Spark
+## sparksftpTest is a project to showcase an ETL pipeline: SFTP - Hive - SFTP with Spark
+
+The process can be summarized in
+
+-   Ingestion of the Data (CSV) from an SFTP server.
+-   Processed by Spark and stored in Parquet/Hive, original data + the ones with tranformations
+-   Storing the results on the SFTP server.
+
+There is currently two implemenations
```
Comment (juanpedromoreno) [inline README.md]:
```suggestion
There are currently two implementations:

- Spark with a SFTP connector
- Spark with Hadoop's SFTP FileSystem Support
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+
+### Spark with a SFTP connector
+
+This is one of the prototypes developed that works and is based on the springml connector https://github.com/springml/spark-sftp
+
+This solution is currently the fastest
+[ReadingSFTPConnectorApp](https://github.com/47deg/sparksftpTest/blob/master/src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPConnectorApp.scala)
```
Comment (juanpedromoreno) [inline README.md]: These paths can be relative: `/src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPConnectorApp.scala`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,48 @@
...
+
+### Spark with Hadoop's SFTP FileSystem Support
+
+[ReadingSFTPHadoopApp](https://github.com/47deg/sparksftpTest/blob/master/src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPHadoopApp.scala)
```
Comment (juanpedromoreno) [inline README.md]: Same here

File: README.md (hunk 2 above)
Comment (vejeta) [reply README.md]: Changed. Thanks

**Review by juanpedromoreno — APPROVED** [review]

**Pull request merged**